src/sentry/api/bases/doc_integrations.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework.request import Request
+
+from sentry.api.base import Endpoint
+from sentry.api.permissions import SentryPermission
+from sentry.api.validators.doc_integration import METADATA_PROPERTIES
+from sentry.auth.superuser import is_active_superuser
+from sentry.utils.json import JSONData
+
+
+class DocIntegrationsPermission(SentryPermission):
+    def has_permission(self, request: Request, view: Endpoint):
+        if not hasattr(request, "user") or not request.user:
+            return False
+
+        if is_active_superuser(request) or request.method == "GET":
+            return True
+
+        return False
```

Comment:
I'm not super well-versed in the auth flow, but I believe this makes the endpoint public, since unknown requests do have the user property set as AnonymousUser. 

If we want to have a scope map and limit it to `member` permissions, it'll be a bit of a refactor, since this is not an `organization` endpoint, and can't just call `has_object_permission(_,_,_,organization)` easily. It'll probably be a abstraction of [this code](https://github.com/getsentry/sentry/blob/master/src/sentry/api/bases/sentryapps.py#L103-L167) to identify an organization in order to call `determine_access` and get scopes

Reply:
All that would just determine that the user belongs to an organization, but `DocIntegrations` don't really care about which organization you are from, since they're universal. Let me know any thoughts you have, @reviewers